ambry-tools/src/main/java/com.github.ambry/clustermap/HelixVcrPopulateTool.java

```diff
@@ -0,0 +1,183 @@
+    System.out.println("Cluster " + destClusterName + " create done.");
+    return true;
+  }
+
+  /**
+   * Update dest cluster information based on src cluster. Dest cluster resource will be recreated if src cluster has any change.
+   * @param srcZkString the src cluster's zk string
+   * @param srcClusterName the src cluster's name
+   * @param destZkString the dest cluster's zk string
+   * @param destClusterName the dest cluster's name
+   */
+  static void updateResourceAndPartition(String srcZkString, String srcClusterName, String destZkString,
+      String destClusterName, boolean dryRun) {
+
+    HelixAdmin srcAdmin = new ZKHelixAdmin(srcZkString);
+    Set<String> srcResources = new HashSet<>(srcAdmin.getResourcesInCluster(srcClusterName));
+    HelixAdmin destAdmin = new ZKHelixAdmin(destZkString);
+    Set<String> destResources = new HashSet<>(destAdmin.getResourcesInCluster(destClusterName));
+
+    for (String resource : srcResources) {
+      if (ignoreResourceKeyWords.stream().anyMatch(resource::contains)) {
+        System.out.println("Resource " + resource + " from src cluster is ignored");
+        continue;
+      }
+      boolean createNewResource = false;
+      if (destResources.contains(resource)) {
+        // check if every partition exist.
+        Set<String> srcPartitions = srcAdmin.getResourceIdealState(srcClusterName, resource).getPartitionSet();
+        Set<String> destPartitions = destAdmin.getResourceIdealState(destClusterName, resource).getPartitionSet();
+        for (String partition : srcPartitions) {
+          if (!destPartitions.contains(partition)) {
+            if (dryRun) {
+              System.out.println("DryRun: Drop Resource " + resource);
+            } else {
+              // This resource need to be recreate.
+              destAdmin.dropResource(destClusterName, resource);
+              createNewResource = true;
+              break;
+            }
+          }
+        }
+      } else {
+        createNewResource = true;
+      }
+      if (createNewResource) {
+        // add new resource
+        Set<String> srcPartitions = srcAdmin.getResourceIdealState(srcClusterName, resource).getPartitionSet();
+        FullAutoModeISBuilder builder = new FullAutoModeISBuilder(resource);
+        builder.setStateModel(LeaderStandbySMD.name);
+        for (String partition : srcPartitions) {
+          builder.add(partition);
+        }
+        builder.setRebalanceStrategy(CrushEdRebalanceStrategy.class.getName());
+        IdealState idealState = builder.build();
+        if (dryRun) {
+          System.out.println("DryRun: Add Resource " + resource + " with partition " + srcPartitions);
+        } else {
+          destAdmin.addResource(destClusterName, resource, idealState);
+          destAdmin.rebalance(destClusterName, resource, 3, "", "");
+        }
+      }
+    }
+    System.out.println("Cluster " + destClusterName + " update done.");
```

Comment:
minor: how about `System.out.println("Cluster " + destClusterName + " is successfully updated!");`